cachi2/core/package_managers/rpm/main.py

```diff
@@ -15,7 +16,7 @@
 
 from cachi2.core.config import get_config
 from cachi2.core.errors import PackageManagerError, PackageRejected
-from cachi2.core.models.input import Request
+from cachi2.core.models.input import Request, _ExtraOptions, _SSLOptions
```

Comment:
Naming nitpick: if you import them, then they probably should not start with an underscore.

Reply:
I would argue this is more than a nitpick, since the underscore is what denotes that a member of a module is private.

Reply:
Added a new commit https://github.com/containerbuildsystem/cachi2/pull/580/commits/3bf9a96a9edd0c9b79ca20bed6ed8b62b51599cb and fixed `_SSLOptions`